pkg/commands/workload.go

```diff
@@ -124,6 +125,7 @@ func (opts *WorkloadOptions) Validate(ctx context.Context) validation.FieldError
 	errs = errs.Also(validation.DeletableKeyValues(opts.Labels, flags.LabelFlagName))
 	errs = errs.Also(validation.DeletableKeyValues(opts.Annotations, flags.AnnotationFlagName))
 	errs = errs.Also(validation.DeletableKeyValues(opts.Params, flags.ParamFlagName))
+	errs = errs.Also(validation.KeyJsonOrYamlValues(opts.ParamsYaml, flags.ParamYamlFlagName))
```

Comment:
For this bit can you add unit test under workload_test.go(TestValidate)?

Reply:
Test added